src/driver_lirc.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016 <Admobilize>
+ * MATRIX Labs  [http://creator.matrix.one]
+ * This file is part of MATRIX Creator MALOS
+ *
+ * MATRIX Creator MALOS is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <iostream>
+#include "./driver_lirc.h"
+#include "./src/driver.pb.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+namespace matrix_malos {
+
+bool LircDriver::ProcessConfig(const DriverConfig& config) {
+  LircParams lirc(config.lirc());
```

Comment:
Perhaps what we need here is a command line argument for MALOS? That says where the binary is? And then we run the command using this binary. But a sanity check for the arguments would be needed before calling system. I don't know the form of the arguments, perhaps they are simple enough for validation.